packages/babel-core/src/transformation/file/options/option-manager.js

```diff
@@ -223,14 +223,21 @@ export default class OptionManager {
   }
 
   mergePresets(presets: Array<string | Object>, dirname: string) {
+    const presetsLoc = presets.map(val =>
+      (typeof val === "string")
+        ? resolve(`babel-preset-${val}`, dirname) || resolve(val, dirname)
+        : val);
+
+    if (presetsLoc.length == 0) {
+      throw new Error("Couldn't find presets at all");
+    }
+
     for (let val of presets) {
       if (typeof val === "string") {
```

Comment:
Should this be changed now since what you added is resolving all presets in the array? It's doing the same thing